changelog.md

```diff
@@ -1,3 +1,9 @@
+# NEXT
+- [FIXED] Fix findOrCreate regression trying to add a transaction even if there is none
+
+# 3.4.1
+- [FIXED] Fix belongs-to-many ambigious id when through model has id
```

Comment:
(It's also a totally insignifanct change why i didn't bother to make it myself :D)